### PR TITLE
README merge deploy kubernetes docs and fix bug

### PR DESCRIPTION
**Pull request by helen-frank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,17 +8,45 @@ Monitore your websites availability, http status code (current and history), cer
...
+
+    * [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+  - Usage
+
+    * `git clone https://github.com/cloudpilot-ai/website-monitoring.git && cd website-monitoring`
+    * Edit `sample/prometheus-targets-cm.yaml` (see targets.yml.example) modify or add `baidu.com` as your desired service address
+    * Create Namespace `kubectl create namespace prometheus`
+    * Starting services `kubectl apply -n prometheus -f sample`
+    * Visualize dashboards http://10.105.212.73:3000/ `kubectl -n prometheus get svc blackbox-exporter` get request address
```
Comment (jwcesign) [inline README.md:30]: I guess using command `kubectl port-forward xxx` is a better choice? then visit `http://locaohost:3000`.

Comment (helen-frank) [reply README.md:30]: ok, add a temporary solution

File: README.md [hunk 2]
```diff
@@ -9,17 +8,54 @@ Monitore your websites availability, http status code (current and history), cer
 * [Prometheus blackbox exporter](https://github.com/prometheus/blackbox_exporter)
 * [Grafana](https://github.com/grafana/grafana)
 
-## Dependencies
+## Deploying
 
-* [docker](https://docs.docker.com/install/)
-* [docker-composer](https://docs.docker.com/compose/install/)
+- Kubernetes
```
Comment (jwcesign) [inline README.md:13]: How about using three-level titles instead of lists? The format maybe look better.

**Comment by jwcesign** [comment]
Other LGTM

File: README.md (hunk 2 above)
Comment (helen-frank) [reply README.md:13]: ok, I'll optimize the layout.

**Pull request merged**